templates/blog-guild.html

```diff
@@ -0,0 +1,55 @@
+{% extends "base.html" %}
+
+{% block content %}
+<section class="p-strip--suru">
+  <div class="row">
+    <div class="col-6">
+      <h1>{% block title %}Blog guild{% endblock %}</h1>
+      <p>A place where people can share ideas and tips, or ask for advice on writing.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="p-divider u-no-margin--bottom">
+        <h2 class="u-sv3">Our blogs</h2>
+        <div class="row">
+          {% for blog in blogs %}
+          <div class="col-4 p-card">
+            <div class="p-media-object--small">
+              <div class="p-media-object__details">
+                <h3 class="p-media-object__title">
+                  {{ blog.Owner }}
+                </h3>
```

Comment:
Should the text be blog title?

Reply:
Hmm... potentially. Decided to use a name for the first draft of this page. Most people just use their name as the blog title anyway.